internal/services/firewall/firewall_data_source.go

```diff
@@ -151,6 +152,26 @@ func firewallDataSource() *pluginsdk.Resource {
 				},
 			},
 
+			"autoscale_configuration": {
+				Type:        pluginsdk.TypeSet,
+				Description: "Properties to provide a custom autoscale configuration to this azure firewall.",
```

Comment:
The description can be removed as this provider doesn't use it to generate the document.